docs/data/joy/components/toggle-button/toggle-button.md

```diff
@@ -0,0 +1,53 @@
+---
+product: joy-ui
+title: React Toggle Button component
+githubLabel: 'component: toggle button'
+---
+
+# Toggle Button
+
+<p class="description">A Toggle Button can be used to group related options.</p>
```

Comment:
I have a question about terminology: What does the term "toggle button" refer to? This document seems to use it as a synonym for both "a group of buttons (that the user can toggle between)" and "the individual button that's part of a group." That can quickly get confusing for the reader. The issue is similar to what we've encountered with the Radio Group. 

I would propose following Radix's terminology:
- a Toggle (Button) is a two-state button which may or may not be part of a group
- a Toggle Group is the component that wraps related Toggle Buttons

If we go with this, here's how I'd phrase the description:

```suggestion
<p class="description">A toggle is a two-state button that may be part of a group of mutually exclusive options.</p>
```

Reply:
Sounds good to me. "A Toggle Button can be used to group related options" is too broad.